sssd_test_framework/hosts/base.py

```diff
@@ -296,3 +296,69 @@ def __init__(self, *args, **kwargs) -> None:
 
         self.fs: LinuxFileSystem = LinuxFileSystem(self)
         self.svc: SystemdServices = SystemdServices(self)
+        self._os_release: dict = {}
+        self._distro_name: str | None = None
+        self._distro_major: int | None = None
+        self._distro_minor: int | None = None
+
+    def _distro_information(self):
+        """
+        Pulls distro information from a host from /ets/os-release
+        """
+        self.logger.info(f"Detecting distro information on {self.hostname}")
+        os_release = self.ssh.run("cat /etc/os-release", log_level=SSHLog.Error).stdout_lines
+        for line in os_release:
+            if "=" not in line:
+                continue
+            k, v = line.rstrip().split("=", maxsplit=1)
+            self._os_release[k] = v.strip('"')
```

Comment:
`dict(csv.reader(os_release.splitlines(), delimiter="="))`

Reply:
This would be a more robust one liner. Just import the csv library that is built-in python.

Reply:
Ack, will do that.